test/webserver.mjs

```diff
@@ -302,7 +297,7 @@ class WebServer {
     });
 
     const extension = path.extname(filePath).toLowerCase();
-    const contentType = mimeTypes[extension] || defaultMimeType;
+    const contentType = MIME_TYPES[extension] || DEFAULT_MIME_TYPE;
```

Comment:
Given that we repeat this code multiple times, would a helper method make sense?
E.g. something like this:
```javascript
#getContentType(filePath) {
  const extension = path.extname(filePath).toLowerCase();
  return MIME_TYPES[extension] || DEFAULT_MIME_TYPE;
}